python/GafferImageTest/OpenImageIOReaderTest.py

```diff
@@ -411,9 +414,10 @@ def testMissingFrameMode( self ) :
 		# the windows match frame 3, but everything else is default
 		reader["missingFrameMode"].setValue( GafferImage.OpenImageIOReader.MissingFrameMode.Black )
 		with context :
+			defaultFormat = formatPlug.getDefaultFormat(context)
```

Comment:
Does this actually get used? I don't see it anywhere in the diff, but perhaps I'm being dumb?